src/js/script.js

```diff
@@ -0,0 +1,270 @@
+    console.log(id);
+    e.stopPropagation();
+
+}
+
+
+//=================================================================================================================
+//=================================================================================================================
+
+//JS for Login Form Validation
+
+function ValidateLogin() {
+    var name = document.getElementById('email').value;
+    var password = document.getElementById('psw').value;
+
+
+    var reg = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/;
+    var passw = /^(?=.*\d)(?=.*[a-z])(?=.*[A-Z])(?=.*[^a-zA-Z0-9])(?!.*\s).{6,14}$/;
+    var statusemail = false;
+    var statuspassword = false;
+    if (name == '') {
+        document.getElementsByClassName("error-email")[0].innerHTML = "Please enter email Id";
+        statusemail = false;
+    } else if (name != '') {
+
+        if (reg.test(name) == false) {
+
+            document.getElementsByClassName("error-email")[0].innerHTML = "Invalid email-id";
+            statusemail = false;
+        } else {
+
+            document.getElementsByClassName("error-email")[0].innerHTML = "";
+            statusemail = true;
+        }
+    }
+
+    if (password == '') {
+        document.getElementsByClassName("error-psw")[0].innerHTML = "Please enter password";
+        //console.log(document.getElementsByClassName("error-email").value);
+        statuspassword = false;
+    } else if (password != '') {
+
+        if (passw.test(password) == false) {
+            document.getElementsByClassName("error-psw")[0].innerHTML = "Invalid password (Password should contain 6 to 14 characters which contain at least one lowercase letter, one uppercase letter, one numeric digit, and one special character)";
+            statuspassword = false;
+        } else {
+            document.getElementsByClassName("error-psw")[0].innerHTML = "";
+            statuspassword = true;
+        }
+    }
+
+    if (statusemail == true && statuspassword == true) {
+        return true;
+    } else {
+        return false;
+    }
+
+}
+
+//=============================================================================================================
+//==============================================================================================================
+
+//JS used for register validation
+
+function ValidationRegister() {
+    var email = document.getElementById('email').value;
+    var password = document.getElementById('psw').value;
+    var fname = document.getElementById('fname').value;
+    var lname = document.getElementById('lname').value;
+    var cnfpassword = document.getElementById('psw-repeat').value;
+
+    var reg = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/;
+    var passw = /^(?=.*\d)(?=.*[a-z])(?=.*[A-Z])(?=.*[^a-zA-Z0-9])(?!.*\s).{6,14}$/;
+    var statusemail = false;
+    var statuspassword = false;
+    var statusfname = false;
+    var statuslname = false;
+    var cnfpsw = false;
+
+    if (fname == '') {
```

Comment:
if a generic form implementation would have been done, you need not to write validations every single time. Too redundant